_i18n/en.yml

```diff
@@ -122,7 +122,7 @@ security_page:
       For more information, please visit [Help](site.baseurl/help) or [contact us](site.baseurl/contact). You can also visit our [open-source repository](https://github.com/18F/identity-idp){:target="_blank"}.
 contact_page:
   content: |
-    First, check out answers to [common questions](site.baseurl/help/). Still have a question or a problem with the platform? Contact us at [login-contact@gsa.gov](mailto:login-contact@gsa.gov).
+    First, check out answers to [common questions](site.baseurl/help/). Still have a question or a problem with the platform? Contact us at our toll-free number 844-875-6446 8:00 a.m. to 8:00 p.m. ET, Monday to Friday (except federal holidays) for additional answers to common questions and to ask questions and speak with an information specialist.
```

Comment:
The last sentence looked a little long so I checked it out with readable.io. Looks like it's at grade level 15 (e.g. college-level). Our interface content should be at a grade 8 reading level. The advice the site gave me was to break the sentence up and use shorter words, but YMMV!